### ISSUE TITLE
Change to minimum focus area of SC 2.4.11 allows for small areas  increases with no contrast

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#1646]
Changes to the minimum focus indicator area now allows for 4px wide along shortest side with a 2px thickness and since it's 2px wide it does not have to meet adjacent contrast.  If this indicator is not inside (color swap) of element and is on the border/outside it effectively allows for an author to add 2px border along one or 2 side such as the bottom that meets the size with no contrast.  Although it does make the control slightly larger on one side- my concern is that this minimum will not be detected by users with low vision.

Previously the idea was the a larger area would be acceptable in lieu of contrast change.  Before the changes to minimum area this would have typically mandated a size increase that was more prevalent either larger on the shortest side 8px or completely around the control 2px.

Code pen: https://codepen.io/mraccess/pen/BaQRooQ

**Comment by alastc** [comment]
Hi Jon,

Your examples fail the "change of contrast" bullet due to the light grey on white background.

I recreated that to meet the contrast and with a couple more examples (and less jumpy, more realistic) here:
https://codepen.io/alastc/pen/JjbNEKz

You'll notice that the (identical from a contrast point of view) examples 3 & 4 are more visible. It's a difficult thing to pin down, but particularly in these grey-scale examples they are harder to see.

Also, the last example (5) fails the size requirement because it is slightly less wide at 32 x 22 than the other examples at 48 x 22. (A 2px border needs to be double the short edge, i.e. at least 44px long along the bottom/top.)

Overall, using the size calculation aimed at the shorter-edge on the longer edge is quite risky, as if you have a bit less text (and the width is not at least twice the height) it fails the size requirement.

So far, my experience has been that it's complicated enough that most people want a simple answer, which is what I'm hoping the top of the understanding doc provides: Use a 2px thick outline!

**Comment by mraccess77** [comment]
Hi @alastc I assumed that the change clause wouldn't apply because the change is not to the control - the change is over the background of the page.  I'll change my examples to black to solve that - but I'd like to know  if the change is over the page background if it makes a difference.

Your example 4 and 5 are easier to see as they use color.  My examples only use size difference with no color or contrast change.

My example 1 is particularly hard for me to see.  The focus indicator is 2px on the 2px on the right - so 4px total - but no color or contrast change.

Thanks for discussing this further.

**Comment by alastc** [comment]
Hi Jon,

Just to try and be clear about the terminology for the SC, referring to my examples:

- Examples 1 and 2 (with the darker grey) do meet "change of contrast", and they would meet non-text contrast because they compare white and dark grey.
- Examples 1 and 2 do not meet "adjacent contrast", but do have the thickness to meet the other half of that bullet. 

So where you talk about size difference, there is contrast there, just not with the element.

We could up the thickness requirement for the size sub-bullet, but if you look around at some of the other examples we've tackled (e.g. #1329), you might find these are unusual niche cases.

**Comment by alastc** [comment]
Hi Jon, I think this issue should be resolved when we discuss #1684 next week, it's in the [survey](https://www.w3.org/2002/09/wbs/35422/wcag22-focus-appearance-updates/).

**Comment by alastc** [comment]
Hi Folks, given that we agreed with the current (slightly updated) text, I assume it is ok to close this one?
Please re-open if not...

**Comment by mraccess77** [comment]
I don't think the proposal resolves this issue - you can still have non-contiguous indicators that are faint but 2 pixels wide on the outset - (or inset but they'd have to contrast with unfocused state).  But I'm willing to drop the issue if it will help to get something passed.

**Comment by alastc** [comment]
Hi Jon,

Thank you, I'd like to get the wider review going.

On the substance, we did discuss increasing the size and/or thickness but didn't get support for that. 

Please keep an eye for this sort of thing in the wild, we can play with test cases, but real-world issues would be more persuasive.
